tools/fuzz/fuzz.cpp

```diff
@@ -673,6 +673,16 @@ void DumpTransformationsBinary(
   transformations_file.close();
 }
 
+// The Chromium project applies the following patch to the protobuf library:
+//
+// source.chromium.org/chromium/chromium/src/+/main:third_party/protobuf/patches/0003-remove-static-initializers.patch
+//
+// This affects how Status objects must be constructed. This method provides a
+// convenient way to get the OK status.
+google::protobuf::util::Status GetProtobufOkStatus() {
```

Comment:
Does this mean that this code will only work with the Chromium patched version of protobuf?

If so, can you please be explicit in this comment about that, I suspect at some point in the future someone might get caught on that.

Reply:
Actually the code is designed to work both with and without the patch. Without the patch there is a redundant call to a copy constructor, but it is functionally correct (and not performance-critical). I have added a comment to explain this.